fastplotlib/graphics/image.py

```diff
@@ -65,20 +72,29 @@ def __init__(
 
         super().__init__(*args, **kwargs)
 
-        self.data = ImageDataFeature(self, data)
+        data = to_gpu_supported_dtype(data)
+
+        # TODO: we need to organize and do this better
+        if isolated_buffer:
+            # initialize a buffer with the same shape as the input data
+            # we do not directly use the input data array as the buffer
+            # because if the input array is a read-only type, such as
+            # numpy memmaps, we would not be able to change the image data
```

Comment:
so, the point of having the option for using an isolated buffer is to be able to change data in the event that the type of the data is a memmap instead of an array?